lib/components/Button.js

```diff
@@ -59,9 +53,16 @@ const Button = React.forwardRef((props, ref) => {
     };
   }
 
-  let Icon =
+  const handleClick = (e) => !loading && !disabled && onClick(e);
```

Comment:
This will result in assigning a boolean value a function if the loading is true, which is not right. Please revert the change.